samples/SoundBoard/src/main/cpp/Synth.h

```diff
@@ -54,9 +54,13 @@ class Synth : public IRenderableAudio, public ITappable {
         }
     }
 
-    void tapSource(bool isOn, int32_t source) {
-        mOscs[source].setWaveOn(isOn);
-    };
+    void setNoteOff(int32_t noteIndex) {
+        mOscs[noteIndex].setWaveOn(false);
```

Comment:
I just realized that you are not using a voice allocator. That may be beyond the scope of this example. Or we can consider it for a later PR.

Reply:
What is a voice allocator? If you mean dynamically allocating voices instead of using an array, we can do this in another PR. This one is getting unwieldy